tests/conftest.py

```diff
@@ -74,6 +82,20 @@ def user(variables):
     return _user(variables)
 
 
+@pytest.fixture(scope='session')
+def entities():
+    return {
+        'organization_label': 'Test_Organization',
+        'activation_key': 'Test AK',
+        'product': 'Test Product',
+        'product_label': 'Test_Product',
+        'yum_repository': 'Zoo',
+        'yum_repository_label': 'Zoo',
+        'errata_id': 'RHEA-2012:0055',
+        'container_repository_label': 'foremanbusybox'
+    }
```

Comment:
I'm debating if these should rather be individual fixtures. I can't quite make up my mind about them. Time will tell whether this is a good choice or if we want to change it at some point. A typical example where we're still learning the best practices.